README.md

```diff
@@ -107,7 +107,8 @@ The complete [role description of a CASA volunteer](https://pgcasa.org/volunteer
 
 **node.js**
 1. (Recommended) Install [nvm](https://github.com/nvm-sh/nvm#installing-and-updating), which is a **n**ode **v**ersion **m**anager.
-1. Install a current LTS version of Node. lts/fermium works.
+1. Install a current LTS version of Node. Running `nvm install` from this directory will read the `.nvmrc` file to install the correct version (lts/iron).
+    - If you are already using asdf, see [legacy file support](https://asdf-vm.com/manage/configuration.html#legacy-version-file) to utilize the .`nvmrc` file. `.tool-versions` is also supported, but may be a bit behind the latest LTS version.
```

Comment:
This bit doesn't work super nicely https://github.com/asdf-vm/asdf-nodejs?tab=readme-ov-file#partial-and-codename-versions

You need an rc file and to set an ENV variable.

Reply:
Good find. Okay, asdf sure is opinionated! Needing an ENV var sort of makes it a per-machine decision rather than per-project decision, and that's a bummer. We should avoid that I think.

Choices:
1. Pin the version to a patch level in .nvmrc, let asdf use .nvmrc via .asdfrc as linked here:
  - -- we'll need to update node/the .nvmrc more often
  - +++ we will have one acceptable node version, no variation
  - +: asdf users will be synced up, where previously they were 2 versions ahead
2. Delete this from README, leave .tool-versions as node version source for asdf:
  - ++: simple/closest to where we were at
  - +/- asdf users will be slightly behind, but at least not way ahead as before
  - ++ less updating node versions manually
3. Leave lts/codename, take node out of .tool-versions, make asdf users mess with their ENV to be able to read node version:
  - --- I wouldn't like doing it, lot of setup involved to get started

Reply:
(I think I like #2, but #1 is fine with me)

Reply:
@elasticspoon I looked at installing specific versions for alpine/Dockerfile for a short while and got frustrated -- I am not sure how to guarantee a specific version there with the current docker image setup. [This discussion](https://gitlab.alpinelinux.org/alpine/abuild/-/issues/9996) also mentions alpine/apk will remove package versions, which made me nervous. I'm not sure you can guarantee an exact version short of using nvm or grabbing a binary.  I undid my change to the dockerfile.

I also removed the asdf bit from the readme, since it is misleading. 

Current state of this PR at least moves in the right direction. What do you think of taking this baby step and thinking more on how to ensure a pinned version on all environments later?